gmso/external/convert_hoomd.py

```diff
@@ -1107,18 +1107,13 @@ def _parse_dihedral_forces(
         },
     }
 
-    if int(hoomd_version[0]) >= 4 or (
-        int(hoomd_version[0]) == 3 and int(hoomd_version[1]) >= 8
-    ):
-        v_hoomd = "gt3.8"
+    if float(f"{hoomd_version[0]}.{hoomd_version[1]}") >= 4.5:
         dtype_group_map["PeriodicTorsionPotential"] = {
             "container": hoomd.md.dihedral.Periodic,
             "parser": _parse_periodic_dihedral,
         }
 
-    else:
-        v_hoomd = "lt3.8"
-        # Should this be periodic, deprecated starting from 3.8.0
+    else:  # Hoomd < 4.5
```

Comment:
Do we want to raise a GMSOError here if HOOMD < 4.0? Since essentially, as it's not a hard dependency, a user could install GMSO with HOOMD 3.0 since it's not in environment.yml, there wouldn't be any compatibility errors.

Reply:
Yeah, I agree. I put an error message at the beginning of `to_gsd_snapsot`, `to_hoomd_snapshot` and `to_hoom_forcefield`